src/components/ContextDropdown.js

```diff
@@ -156,7 +156,7 @@ function ContextDropdown ({ onItemChosen, selectedContext, showRequestButton = f
               padding: '15px',
               textAlign: 'center'
             }}
-            >It looks like you currently do not have access to any contexts matching "{inputValue}". Please request access below.</div>
+            >It looks like you currently do not have access to any contexts matching “{inputValue}”. Please <Link to="/request" target="_blank" rel="noopener noreferrer">/request</Link> access below.</div>
```

Comment:
why link to /request here? request is referring to the request button below., not the request form. will probably confuse a few users.

Reply:
Oh, okay. Can be reverted.